internal/core/items/text.rs

```diff
@@ -324,11 +325,12 @@ impl Item for TextInput {
                             return KeyEventResult::EventAccepted;
                         }
                     },
+                    Some(_) => { return KeyEventResult::EventIgnored; }
```

Comment:
This block needs re-formatting (as indicated by the CI).

Reply:
I dont't know how to run cargo on github repo can you do it instead